### ISSUE TITLE
Problem if text contains 5-sign

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halamalala/inkscape2tikz#35]
```
What steps will reproduce the problem?
1. any text that contains a %-sign

What is the expected output? What do you see instead?
%-sign in a text should be converted to \% in the tex-file otherwise the rest 
of that line will be interpreted as a comment and therefore results in an error

What version of the product are you using? On what operating system?
Downloaded a couple of days ago.
```

Original issue reported on code.google.com by `e.blechs...@gmail.com` on 20 Oct 2011 at 11:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `kjel...@gmail.com` on 15 Jul 2012 at 8:03
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Currently the exporter interprets all text as raw TeX. This is useful in some 
cases, but should not be the default behavior. See also issue 2. 
```

Original comment by `kjel...@gmail.com` on 15 Jul 2012 at 8:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision 097126e42d4e
```

Original comment by `kjel...@gmail.com` on 15 Jul 2012 at 9:40
- Changed state: **Fixed**
